physics/smoke_dust/dust_data_mod.F90

```diff
@@ -76,12 +62,16 @@ module dust_data_mod
        0.50,    & ! Silty Clay      - 11
        0.45,    & ! Clay            - 12
        9999.0 /)   ! Other           - 13
-  ! -- FENGSHA uses precalculated drag partition from ASCAT. See: Prigent et al. (2012,2015)
-  integer, parameter :: dust_calcdrag = 1
 
-  real(kind_phys) :: dust_alpha = 2.2
+  ! -- FENGSHA uses precalculated drag partition
+  integer, parameter :: dust_calcdrag = 1
+  ! -- FENGSHA dust moisture parameterization 1:fecan  -  2:shao 
+  integer :: dust_moist_opt = 1
+  
+  real(kind_phys) :: dust_alpha = 1.0
```

Comment:
Should these be protected with `parameter`?

Reply:
These are tunable with input namelist options.